### DISCUSSION TITLE
Raspberry pi 4 - Production

### DISCUSSION THREAD

**Discussion by darrenleesmith** [revoxhere/duino-coin#1490] [General]
Hello,

I have a Raspberry Pi 4 8GB (Running a 64 bit version of Ubuntu (Possibly 21,04)) When I first started mining duino coins on version 2.7 I was probably making 50/60 coins a day at 0.9MH/s 

But now running version 3.1 I am running at 0.73MH/s earning 1.95 duino coins a day with a 64% kolka drop.

I have tried the fast hash and was running at 5/6 MH/s but only earning 0.5 coins.

Am I doing something wrong? I see people on the Facebook group with 5 Arduinos getting 60/70 coins a day and I am getting basically nothing, I have tried dropping the intensity and cores but it doesn't really change anything.

![coin](https://user-images.githubusercontent.com/48777290/165982039-034b3fa6-bea3-4497-b10e-1b46af1784d5.PNG)

**Comment by achapkowski** [comment]
I see the same issue

**Comment by Grantrocks** [comment]
This is because you are using 4 threads. I would use 2 threads. Thats what i use with my raspberry pi.
A little info on this below
![image](https://user-images.githubusercontent.com/46384149/167644527-755e917b-7caf-4c78-982b-b1d072d7e3e1.png)

**Reply by Grantrocks** [reply]
i would also use the latest version because that detects that you are using a raspberry pi

**Reply by darrenleesmith** [reply]
This second my settings are (using 3.18):

### Current - 3.3 duino a day (100% kulka) - 1.05 Mh/s

[PC Miner]
username = [Removed]
mining_key = [Removed]
intensity = 90
threads = 4
start_diff = LOW
donate = 1
identifier = Pi8G
algorithm = DUCO-S1
language = english
soc_timeout = 15
report_sec = 300
discord_rp = n
 
I have changed it to 2 threads, but the ping is really high at the moment, I will update later, how many coins are you mining a day?

**Reply by Grantrocks** [reply]
About 15 17

**Reply by Grantrocks** [reply]
And the reason the people in Facebook are getting so much is because areuinos get the highest reward

**Reply by darrenleesmith** [reply]
> About 15 17

Which PI do you have? I have changed it to 2 treads and I'm still getting 3.3

I have also tried doing it on a PICO, but this version is having an issue with PICO, I might have to bite the bullet and buy some Arduino's

**Reply by Grantrocks** [reply]
How many devices do you have mining. I only have a arduino and a raspberry pi 4 4gb overclocked to 2.2ghz on a 64bit raspberry pi os. Not compiled libducohasher.

**Reply by Grantrocks** [reply]
my settings
threads: 2
intensity: 95
start_diff=MED
these are the only ones different from you
